cob_object_detection_msgs/CMakeLists.txt

```diff
@@ -50,6 +50,7 @@ add_service_files(
     AcquireObjectImage.srv
     BagTrainObject.srv
     BaTestEnvironment.srv
+    ComputeGraspsVacuumGripper.srv
```

Comment:
should end up in either a manipulation package or a scenario specific package. There is no reason why this scenario package cannot be shared for different use cases.